tutor/src/models/user/menu.js

```diff
@@ -115,11 +115,8 @@ const ROUTES = {
 
 };
 
-const TUTOR_HELP = 'http://openstax.force.com/support?l=en_US&c=Products%3ATutor';
-const TUTOR_CONTACT = 'http://openstax.force.com/support/?cu=1&fs=ContactUs&l=en_US&c=Products%3ATutor';
-const CONCEPT_COACH_HELP = 'http://openstax.force.com/support?l=en_US&c=Products%3AConcept_Coach';
-const CONCEPT_COACH_CONTACT = 'http://openstax.force.com/support/?cu=1&fs=ContactUs&l=en_US&c=Products%3AConcept_Coach';
-
+const TUTOR_HELP = 'https://openstax.secure.force.com/help';
+const TUTOR_CONTACT = 'https://openstax.org/contact';
```

Comment:
Heh maybe the code could become more DRY if we used those constants for the links above?